Kitodo/src/main/java/de/sub/goobi/forms/IndexingForm.java

```diff
@@ -843,22 +854,13 @@ private void resetGlobalProgress() {
         indexedFilter = 0;
     }
 
-    private void enableSortingByTextField(String type, String field) {
-        try {
-            indexRestClient.initiateClient();
-            indexRestClient.setIndex(ConfigMain.getParameter("elasticsearch.index", "kitodo"));
-            indexRestClient.enableSortingByTextField(type, field);
-        } catch (CustomResponseException | IOException e) {
-            logger.error(e);
-        }
+    private static String readMapping() throws IOException, ParseException {
+        JSONParser parser = new JSONParser();
+        ClassLoader classloader = Thread.currentThread().getContextClassLoader();
+        InputStream inputStream = classloader.getResourceAsStream("mapping.json");
```

Comment:
You are open a stream object without closing it never. Maybe you can use the try-with-resources statement here and avoid / handle the unhandled IOException here.